tools/android/android_sdk_repository_template.bzl

```diff
@@ -210,6 +218,11 @@ def create_android_sdk_rules(
         actual = ":org_apache_http_legacy-%d" % default_api_level,
     )
 
+    native.alias(
+        name = "android_car",
+        actual = ":android_car-%d" % default_api_level,
+    )
```

Comment:
this should probably go inside the `if` above so that the `all` and `...` target patterns still work

Reply:
Good point, I'll move it. Perhaps we should also consider doing the same thing for `org_apache_http_legacy`.

Reply:
@ahumesky could you take a look again?